soh/src/code/z_demo.c

```diff
@@ -509,7 +509,52 @@ void Cutscene_Command_Terminator(GlobalContext* globalCtx, CutsceneContext* csCt
         temp = 1;
     }
 
-    if ((csCtx->frames == cmd->startFrame) || (temp != 0) || ((csCtx->frames > 20) && (randoCsSkip || debugCsSkip))) {
+    bool playCutscene = false;
```

Comment:
Add a comment `// HACK: Aligns A/V timing for Credits`

Reply:
done